docs/DeveloperGuide.md

```diff
@@ -376,18 +375,122 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User requests to list orders
-2.  ReadyBakey shows a list of orders
-3.  User requests to clear all saved customers
-4.  ReadyBakey clears all customers
+1. User requests to list orders
+2. ReadyBakey shows a list of orders
+3. User requests to clear all saved customers
+4. ReadyBakey clears all customers
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
 * 2a. The list is empty.
 
-  Use case ends.
+**Use case: Exits the ReadyBakey program**
+
+**MSS**
+
+1. User requests to exit the program.
+2. ReadyBakey says bye and closes the application.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. User closes the program without using CLI.
+    * 1a1. ReadyBakey closes the application.
+
+**Use case: Locate customer by their name**
+
+**MSS**
+
+1. User requests to find the customer by name in ReadyBakey.
+2. ReadyBakey finds the customers whose names match the requested customer.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. ReadyBakey detects no customers that match the user's request.
+  * 1a1. ReadyBakey requests for the correct customer name.
+  * 1a2. User enters the correct customer name.
```

Comment:
Should this be "ReadyBakey returns no results and queries user if the correct customer name has been entered?". There is a possibility that the user is trying to see if the name exists in the data base rather than him typing it wrongly! What are your thoughts?

Reply:
I agree with @Geralddtan on this!